src/batched/sparse/impl/KokkosBatched_Spmv_Team_Impl.hpp

```diff
@@ -0,0 +1,155 @@
+#ifndef __KOKKOSBATCHED_SPMV_TEAM_IMPL_HPP__
+#define __KOKKOSBATCHED_SPMV_TEAM_IMPL_HPP__
+
+
+/// \author Kim Liegeois (knliege@sandia.gov)
+
+#include "KokkosBatched_Util.hpp"
+
+namespace KokkosBatched {
+
+  ///
+  /// Team Internal Impl
+  /// ==================== 
+  struct TeamSpmvInternal {
```

Comment:
Why is this not `TeamThreadSpmvInternal`? The other one is `TeamVectorSpmvInternal`

Reply:
I followed the same naming logic as other KK batched kernels but I can revisit that without any issue.